common/src/main/kotlin/com/hartwig/actin/trial/input/FunctionInputResolver.kt

```diff
@@ -993,6 +1005,10 @@ class FunctionInputResolver(
         return OneProteinOneString(proteinName = parameterAsString(function, 0), string = parameterAsString(function, 1))
     }
 
+    private fun toTreatmentCategoriesSet(input: Any): Set<TreatmentCategory> {
+        return toStringList(input).map{TreatmentCategoryResolver.fromString(it)}.toSet()
```

Comment:
There should be spaces around curly braces here, or you could use a reference:
```suggestion
        return toStringList(input).map(TreatmentCategoryResolver::fromString).toSet()
```